### DISCUSSION TITLE
Announce: Helper library for writing IAM Policy Documents

### DISCUSSION THREAD

**Discussion by JackKelly-Bellroy** [jcouyang/dhall-aws-cloudformation#19] [General]
(Please close/delete this thread if you feel it's inappropriate)

At Bellroy, we've made heavy use of `dhall-aws-cloudformation` to simplify our CloudFormation templates. Writing Dhall was much easier than writing raw YAML, and using this library was much better still. Thank you very much.

Writing IAM Policy Documents using Dhall's native JSON type was annoying, and so we developed an internal library to make it much cleaner to write them. Before Christmas last year, we open-sourced this library in the hope that it will be useful to others.

https://github.com/bellroy/dhall-aws-iam-policy-document/

**Comment by jcouyang** [comment]
Yeah the JOSN policy is a pain to me as well but luckily we dont have that many police docs to maintain

This looks pretty promising I may try to rewrite our policy doc with it. Great work thanks
